pkg/multiTenancyPlugins/authorization/defaultAuthZImpl.go

```diff
@@ -50,6 +50,9 @@ func (defaultauthZ *DefaultAuthZImpl) Handle(command utils.CommandEnum, cluster
 				return err
 			}
 			oldconfig.ContainerConfig.Config.Labels[headers.TenancyLabel] = r.Header.Get(headers.AuthZTenantIdHeaderName)
+			if err := hostFSMountCheck(r.Header.Get(headers.AuthZTenantIdHeaderName), oldconfig.HostConfig.Binds); err != nil {
```

Comment:
let's remove this test for now as we allow many arguments that should be disabled for multi tennat. Let's move all of these checks to another place